apps/fhir/bluebutton/tests/test_models.py

```diff
@@ -78,27 +70,27 @@ def test_immutable_fhir_id(self):
         with self.assertRaises(ValidationError):
             cw.fhir_id = "-20000000000002"
 
-    def test_immuatble_user_hicn_hash(self):
+    def test_muatble_user_hicn_hash(self):
```

Comment:
Minor detail, but is this mutable? thinking this is just a typo. Same below on line 84.

Reply:
> Minor detail, but is this mutable? thinking this is just a typo. Same below on line 84.

@oragame I renamed the test methods, since hicn/mbi hash values can now be changed. Before they could not.

Reply:
Makes sense but I think there is a typo here.

Reply:
> Makes sense but I think there is a typo here.

@oragame Very sorry :man_facepalming: .  I read your comment too quickly and didn't see the actual typo before responding :smile:   Also didn't notice it in the original code.   Will fix shortly.   @njdister